flexmeasures/ui/views/analytics.py

```diff
@@ -535,6 +535,16 @@ def make_power_figure(
     else:
         title = "Electricity production from %s" % resource_display_name
 
+    resolution_str = "?"
+    if data.index.freq is not None:
```

Comment:
I think this should be checking for `data.event_resolution` instead.

Reply:
Actually, we are receiving a `pd.DataFrame` at this point. The return type of `data.queries.analytics.get_power_data` is `Tuple[pd.DataFrame, pd.DataFrame, pd.DataFrame, dict]`, with lines like these:

```
power_df: pd.DataFrame = simplify_index(
       power_bdf, index_levels_to_columns=["belief_horizon", "source"]
)
```
So I don't believe `event_resolution` is available.